_config.yml

```diff
@@ -40,6 +40,7 @@ exclude:
   - Rakefile
   - Gemfile
   - Gemfile.lock
+  - gems/
```

Comment:
I'm not sure if it's something strange about my machine's config, but bundler seems to install gems in `~/.gem`. @PeterJCLaw does it install them in a `gems` directory in the repo for you?